python/ray/serve/_private/http_state.py

```diff
@@ -76,6 +77,16 @@ def set_status(self, status: HTTPProxyStatus) -> None:
         self._status = status
         self.update_actor_details(status=self._status)
 
+    def set_draining_flag(self, node_id: str, draining: bool):
+        """Set the draining flag on the http proxy.
+
+        Set the draining flag on the http proxy. When the flag is set to false, also
+        update status to from HEALTHY to DRAINING to display on the dashboard.
```

Comment:
```suggestion
        update status to from HEALTHY to DRAINING.
```
a bit of an abstraction leak :)